HM_11/hw_15.Task01/Attributes/MyCreditCardAttribute.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+
+namespace hw_15.Task01.Attributes
+{
+    [AttributeUsage(AttributeTargets.Property)]
+    class MyCreditCardAttribute : ValidationAttribute
```

Comment:
looks good, but seems the logic does not work correctly
![image](https://user-images.githubusercontent.com/3023032/129866894-691ece4c-516c-4d67-a9cd-8a3451850026.png)
two attempts - no luck :(

Reply:
VadzimPapko, It is necessary to use real credit card number. 1234567890123456 - is not valid credit card number even if we use standard attribute "CreditCard"